src/components/entity/feed-entry.vue

```diff
@@ -62,6 +62,7 @@ except according to the terms contained in the LICENSE file.
       </template>
       <template v-else-if="entry.action === 'entity.update.version'">
         <span class="icon-pencil"></span>
+        <span class="title">
```

Comment:
To separate the event title from the version, which was also in the `.feed-entry-title`  div, I wound up wrapping it in another span. But only the update event title!